packages/utils/src/lib/transform.ts

```diff
@@ -151,3 +151,11 @@ export function toOrdinal(value: number): string {
 
   return `${value}th`;
 }
+
+export function removeUndefinedAndEmptyProps<T extends object>(obj: T): T {
+  return Object.fromEntries(
+    Object.entries(obj).filter(
+      ([_, value]) => value !== undefined && value !== '',
```

Comment:
I think this is sufficient, but that's nitpick
```suggestion
      ([, value]) => value !== undefined && value !== '',
```